package.json

```diff
@@ -101,7 +112,9 @@
     "stylelint-scss": "^4.2.0",
     "stylelint-webpack-plugin": "^3.2.0",
     "svg-url-loader": "^7.1.1",
+    "ts-node": "^10.7.0",
```

Comment:
Adding ts-node should it be used for other TS compile tasks?

Reply:
We use it to run typescript in node. Which is done in this script, that has been introduced in this PR:
https://github.com/danskernesdigitalebibliotek/dpl-react/pull/18/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R38

Reply:
So it's to speed up local development and have "continuous compilation" of TS?